diff --git a/dist/flowchart-elk-definition-4ac3371a.js b/dist/flowchart-elk-definition-4ac3371a.js
index 1b5ae77fdfc115ebd3c2d2d9d920dba4997c4b76..e566adb9702e1b3971183ff828a021b3d53219b1 100644
--- a/dist/flowchart-elk-definition-4ac3371a.js
+++ b/dist/flowchart-elk-definition-4ac3371a.js
@@ -730,7 +730,7 @@ const draw = async function(text, id, _version, diagObj) {
     }
   });
   insertChildren(graph.children, parentLookupDb);
-  log.info("after layout", JSON.stringify(graph, null, 2));
+  // log.info("after layout", JSON.stringify(graph, null, 2));
   const g = await elk.layout(graph);
   drawNodes(0, 0, g.children, svg, subGraphsEl, diagObj, 0);
   log.info("after layout", g);
